### ISSUE TITLE
Not properly be parsed by PlaceholderAPI

### ISSUE THREAD

**Issue by TheMrEngMan** [Dart2112/AFKPlusPAPI#1]
I can't seem to get it to properly be parsed by PlaceholderAPI.
I've installed the latest version of both AFK+ from the Spigot resource page and AFKPlusPAPI from the GitHub release.
No warnings/errors in console and the base functionality works great!

The issue is the placeholder is not being evaluated. When I run:
`papi parse (any online player) %AFKPlus_Status%`
in the console or in-game, it simply returns:
`%AFKPlus_Status%`
(same with `%AFKPlus_AFKTime%`).

Using Paper #90 1.16.1.

**Comment by TheMrEngMan** [comment]
I've compiled this plugin and added debug print statements. Turns out that the `AFKPlus_` part in the placeholders is removed before being passed to the handler method.
The workaround is to use `%AFKPlus_AFKPlus_Status%` and `%AFKPlus_AFKPlus_AFKTime%` as the placeholders.

I'm keeping the issue open for now since according to this https://github.com/LapisPlugins/AFKPlus/issues/4#issuecomment-629597451 the placeholders are wrong.
